src/main/java/com/github/princesslana/eriscasper/rx/websocket/RxWebSocket.java

```diff
@@ -16,6 +17,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(RxWebSocket.class);
 
+  private volatile boolean closed = false;
```

Comment:
This is to be removed in favor of picking up the Closed web socket event?

Reply:
I need to see where else it's being used, if unused I'll remove it.

Reply:
Yeah it's being removed